### ISSUE TITLE
[Active User - More action]: Screen reader does not announce state as collapsed on More actions menu item.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13204]
### Steps to reproduce:
1.	Open URL and login with valid credentials.
2.	Navigate to 'Dashboard view' by tab key and activate through enter key.
3.	Navigate to 'Navigation menu-> User-> Active user' and activate through enter key.
4.	Navigate to 'More actions' control and activate through enter key.
5.	Verify that screen reader does announce state as collapsed on More action menu item or not.

### Actual behavior:
Screen reader does not announce state as collapsed on More actions menu item.

### Expected behavior:
Screen reader should announce state as collapsed on More actions menu item.
### Additional Note:
Same issue repro on 'Change view' icon control throughout the website.

### System configuration
Application: M365 Admin Portal
URL: https://admin.microsoft.com/Adminportal/Home
Browser: Edge, Version 96.0.1054.62 (Official build) (64-bit)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA Version 2021.1

#### Windows version:
Window 11, Version 21H2 (OS Build 22000.376)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, also reproducible through NVDA version 2020.4

**Comment by feerrenrut** [comment]
This needs further investigation by Microsoft developers. Please provide a minimal test case for us to investigate. I'm closing this issue for now.
